packages/password-field/test/password-field.test.js

```diff
@@ -145,6 +145,14 @@ describe('password-field', () => {
         expect(passwordField.hasAttribute('focus-ring')).to.be.true;
       });
     });
+
+    describe('mouse', () => {
+      it('should not set focus-ring attribute when focusing on mousedown', () => {
+        mousedown(passwordField);
```

Comment:
Does the `mousedown()` call play a role in this test? It seems that removing the `mousedown()` call changes nothing.

Reply:
Thanks, removed and renamed the test accordingly.